src/ngx_http_auth_jwt_module.c

```diff
@@ -293,6 +293,13 @@ static ngx_int_t ngx_http_auth_jwt_handler(ngx_http_request_t *r)
 			jwt_free(jwt);
 		}
 
+		if (!jwtcf->auth_jwt_redirect)
+		{
+			// When no redirect is needed, we can bail out already, because no "Location" header needs to be constructed
+
```

Comment:
Extra line break here. Sorry for the nit-pick. :)

Reply:
Sorry, not sure what you mean? You mean remove the extra line break?

Or two line breaks before the `return`?

Reply:
```

Reply:
It is now at the end of the function, but took it along in my rebase.